install/0000_30_machine-api-operator_13_servicemonitor.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    k8s-app: machine-api-operator
+  name: machine-api-operator
+  namespace: openshift-machine-api
+spec:
+  endpoints:
+    - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+      interval: 30s
```

Comment:
It should be the default scrape interval for Prometheus so there's no need to specify it again.